### PR TITLE
use shows/venues from redux, add ProgressBar component, fix date formatting

### PR DESCRIPTION
**Pull request by doncote**

### CODE REVIEW

File: src/components/identity/UserShows.tsx [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import React, { useContext, useEffect, useState } from 'react'
 import PageHeading from '../shared/PageHeading'
-import { AppContext } from '../../context/AppProvider';
-import { IShow } from '../shows/Show';
-import axios, { AxiosResponse } from 'axios';
-import ListShows from '../shows/ListShows';
-import { LinearProgress, TableContainer, Paper, Table, TableRow, TableCell } from '@material-ui/core';
+import { AppContext } from '../../context/AppProvider'
+import { IShow } from '../shows/Show'
```
Comment (shaunoff) [inline src/components/identity/UserShows.tsx:4]: Types may be better in the stores directory. Doesn't matter that much though.

Comment (doncote) [reply src/components/identity/UserShows.tsx:4]: oh yeah good call.  i'll move those over.

File: src/components/songs/Songs.tsx [hunk 2]
```diff
@@ -1,167 +1,173 @@
...
-			});
-		  }
-	};
+  const data = songs.map((s: ISong) => [
+    [s.slug, s.title],
+    s.author_name,
+    s.cover ? 'cover' : 'original',
+    s.times_played,
+    s.date_last_played,
+  ])
```
Comment (shaunoff) [inline src/components/songs/Songs.tsx]: This can be placed in a selector and memoized. No big deal for now though.

**Review by shaunoff — APPROVED** [review]
couple of places we can optimize but this is a good step forward.

**Pull request merged**